src/ucs/async/async.c

```diff
@@ -14,6 +14,7 @@
 #include <ucs/debug/debug_int.h>
 #include <ucs/datastruct/khash.h>
 #include <ucs/sys/stubs.h>
+#include <ucs/sys/math.h>
```

Comment:
why do we need math.h here?

Reply:
ucs_max needs it. since we remove math.h off of mpmc.h, we need to add math.h here.

Reply:
ok